### DISCUSSION TITLE
Upgrade from cc2531 v1.2 to SONOFF Zigbee 3.0 USB Dongle Plus

### DISCUSSION THREAD

**Discussion by Shaman2** [Koenkk/zigbee2mqtt#10509] [Devices]
I have zigbee cc2531 coordinator usb stick with firmware version 1.2 (20201127)
What I need to do for upgrade to SONOFF Zigbee 3.0 USB Dongle Plus without repairing my 22 devices?

**Comment by Koenkk** [comment]
https://www.zigbee2mqtt.io/guide/faq/#how-do-i-migrate-from-one-adapter-to-another

**Comment by Shaman2** [comment]
but I dont understand I need to copy ieee address of the old adpter into the new or not?

**Reply by Koenkk** [reply]
You need to copy it, since accoridng to https://www.zigbee2mqtt.io/guide/faq/#what-does-and-does-not-require-repairing-of-all-devices you do not require repairing you can migrate without repairing.

**Comment by Shaman2** [comment]
I think it's unnecessary, but maybe I'm wrong?
